locust/main.py

```diff
@@ -38,6 +38,15 @@
 version = locust.__version__
 
 
+# Options to ignore when using a custom shape class without `use_common_options=True`
+# See: https://docs.locust.io/en/stable/custom-load-shape.html#use-common-options")
+IGNORABLE_OPTIONS = {
```

Comment:
How do you feel about renaming this variable to COMMON_OPTIONS?

Reply:
Done 👍🏼 

In fact, I hesitated between the 2 but then I added a docstring to be explicit so `COMMON_OPTIONS` is better

Reply:
My thoughts exactly :)